son-gtkapi/models/vim_manager_service.rb

```diff
@@ -42,53 +42,58 @@ def self.config(url:, logger:)
     @@logger = logger
     @@logger.debug(method) {'entered'}
   end
-    
+
   def self.url
     @@logger.debug(LOG_MESSAGE + "#url") {'@@url='+@@url}
     @@url
   end
-  
+
   def self.find_vims(params)
     method = LOG_MESSAGE + "##{__method__}(#{params})"
-    @@logger.debug(method) {'entered'}    
+    @@logger.debug(method) {'entered'}
     begin
-      response = getCurb(url:url, headers:JSON_HEADERS) 
+      response = getCurb(url:self.url+'/vim', headers:JSON_HEADERS, logger:@@logger)
       @@logger.debug(method) {'response='+response.body}
       JSON.parse response.body
     rescue => e
       @@logger.error(method) {"Error during processing: #{$!}"}
       @@logger.error(method) {"Backtrace:\n\t#{e.backtrace.join("\n\t")}"}
-      nil 
+      nil
     end
   end
-  
+
   def self.create_vim(params)
     method = LOG_MESSAGE + "##{__method__}(#{params})"
     @@logger.debug(method) {"entered"}
-    
+
     begin
-      @@logger.debug(method) {"@url = "+@url}
-      #response = RestClient.post(@url+'/vim', params.to_json, content_type: :json, accept: :json) 
-      response = postCurb(url: @@url+'/vim', body: params.to_json) 
+      @@logger.debug(method) {"@url = " + self.url}
+      response = postCurb(url:self..url+'/vim', body: params.to_json)
       @@logger.debug(method) {"response="+response}
       response
     rescue => e
       @@logger.error(method) {"Error during processing: #{$!}"}
       @@logger.error(method) {"Backtrace:\n\t#{e.backtrace.join("\n\t")}"}
-      nil 
-    end      
+      nil
+    end
   end
-  
+
   def self.find_vim_request_by_uuid(uuid)
     method = LOG_MESSAGE + "##{__method__}(#{uuid})"
     @@logger.debug(method) {'entered'}
     begin
-      response = getCurb(url:@@url+'/vim_request/'+uuid, headers: JSON_HEADERS) 
-      JSON.parse response.body
+      response = getCurb(url:self.url+'/vim_request/'+uuid, headers: JSON_HEADERS)
```

Comment:
(I know this was already implemented as singular, but since you're touching it...) ;-)